Conversion-Class Temperatura

```diff
@@ -0,0 +1,18 @@
+package com.company;
+
+public class Temperatura {
+
+    public double temperatura;
+    public void setTemperatura(double temperaturaZilei) {
+        this.temperatura = temperaturaZilei;
+    }
+    public double getTemperatura(){
+        return temperatura;
+    }
+
+    public void temperatura (double c, double f) {
```

Comment:
The naming of the function should also contain what kind of conversion it is, for example `conversionCelsiusToFahrenheit`, or the other way around, depending on your function purpose